src/components/access_group/AccessGroupCreate.vue

```diff
@@ -268,7 +273,7 @@ export default {
 };
 </script>
 
-<style>
+<style scoped>
```

Comment:
We should be consistent there in the future. We start to have a mix. A follow up thing.